doc/source/ray-overview/installation.rst

```diff
@@ -14,22 +14,67 @@ From Wheels
 You can install the latest official version of Ray from PyPI on Linux, Windows
 and macOS as follows:
 
-.. code-block:: bash
 
-  # Install Ray with support for the dashboard + cluster launcher
-  pip install -U "ray[default]"
 
-  # Install Ray with minimal dependencies
-  # pip install -U ray
+.. tabbed:: Default
 
-To install Ray libraries:
+  To install Ray with default dependencies for the dashboard and cluster launcher:
 
-.. code-block:: bash
+  .. code-block:: bash
+
+    # Install Ray with support for the dashboard + cluster launcher
+    pip install -U "ray[default]"
+
+
+.. tabbed:: Minimal
+
+  To install a minimal version of Ray *without* support for the dashboard and cluster launcher:
+
+  .. code-block:: bash
+
+    # Install Ray with minimal dependencies
+    pip install -U ray
+
+
+.. tabbed:: AI Runtime
+
+  To install Ray with support for Ray AI Runtime (Data, Train, Tune, Serve):
+
+  .. code-block:: bash
+
+    # Install Ray + dependencies for Ray AI Runtime
```

Comment:
should we add something like "without support for the dashboard and cluster launcher"

Reply:
Hmm do you think the tip below is sufficient? I'm worried that it might be confusing if we do too much inclusion/exclusion, and was aiming to be as modular and additive as possible.

Reply:
I guess what I'm not sure about when I read this line is: what does "Ray" refer to here? Is it minimal Ray or?

Reply:
Great point, updated the design and replaced "Minimal" with "Core"!